AmplifyPlugins/API/AWSAPICategoryPluginTests/Reachability/NetworkReachabilityNotifierTests.swift

```diff
@@ -81,7 +76,7 @@ class NetworkReachabilityNotifierTests: XCTestCase {
         }, receiveValue: { value in
             values.append(value.isOnline)
             if values.count == 2 {
```

Comment:
the setup portion of this test is now related to the assert here, from the reader's prespective, i have no idea why you are checking for count 2 unless i look into the set up of the test. even though all tests are using `MockReachability.iConnection = .wifi` at line 29, i think it's clearer if you put that line in every test and move it out of the setup